tests/end2end/test_registering_user.py

```diff
@@ -12,11 +12,17 @@
 
 
 # noinspection PyUnusedLocal
-def test_user_can_register(user, testapp):
+def test_user_can_register(superuser, testapp):
     """Register a new user."""
     old_count = len(User.query.all())
     # Goes to homepage
     res = testapp.get('/')
+    # Fills out login form in navbar
+    form = res.forms['loginForm']
+    form['username'] = superuser.email
+    form['password'] = 'myPrecious'
+    # Submits
+    form.submit().follow()
     # Clicks Create Account button
     res = res.click(_('Create account'))
```

Comment:
After login, a user can't go straight to "register user" form. First they must click to "Users" page, then they can find-and-click the "Create account" button (think this was one of the last things you removed).